reward/staking_manager.go

```diff
@@ -88,21 +87,17 @@ func (sm *StakingManager) GetStakingInfo(blockNum uint64) *StakingInfo {
 	}
 
 	logger.Debug("Get stakingInfo from header.", "blockNum", blockNum, "staking block number", stakingBlockNumber, "stakingInfo", calcStakingInfo)
+	sm.stakingInfoCache.add(calcStakingInfo)
```

Comment:
This code is duplicated with the line 101.

Reply:
Please remove this.

Reply:
Thank you. It is now erased.